openedx_webhooks/templates/github_pr_cover_letter.md.j2

```diff
@@ -0,0 +1,47 @@
+{{ body }}
+
+## [PROJ-XXXX](https://openedx.atlassian.net/browse/PROJ-XXXX)
+
+Thanks for the pull request, @{{ user }}!
+
+Please add a detailed description of your changes here.
```

Comment:
Let's make this a checkbox that mentions having the JIRA ticket number in the description.